pkg/network/ebpf/c/tracer/tracer.h

```diff
@@ -29,6 +29,17 @@ typedef enum {
 
 #define CONN_DIRECTION_MASK 0b11
 
+typedef struct {
+    __u16 chosen_version;
+    __u16 cipher_suite;
+    __u8  offered_versions;
+} tls_info_t;
+
+typedef struct {
+    tls_info_t info;
+    __u64 updated;
+} tls_info_wrapper_t;
```

Comment:
the new struct is copy pasted/inspired from the classification structs
while we do have good reasons for the complexity there, is there any justification for following the same complexity here?

Reply:
basically just that the map leaks. we've used this pattern on NPM as well, I find it to be a nice solution considering `tls_info_t` is eventually embedded into `conn_t` on the NPM side which is highly stack constrained (for now)